docs/_src/api/api/reader.md

```diff
@@ -48,7 +48,7 @@ While the underlying model can vary (BERT, Roberta, DistilBERT, ...), the interf
 #### \_\_init\_\_
 
 ```python
- | __init__(model_name_or_path: str, model_version: Optional[str] = None, context_window_size: int = 150, batch_size: int = 50, use_gpu: bool = True, no_ans_boost: float = 0.0, return_no_answer: bool = False, top_k: int = 10, top_k_per_candidate: int = 3, top_k_per_sample: int = 1, num_processes: Optional[int] = None, max_seq_len: int = 256, doc_stride: int = 128, progress_bar: bool = True, duplicate_filtering: int = 0, use_confidence_scores: bool = True, proxies: Optional[Dict[str, str]] = None, local_files_only=False, force_download=False, use_auth_token: Optional[Union[str,bool]] = None, **kwargs)
+ | __init__(model_name_or_path: str, model_version: Optional[str] = None, context_window_size: int = 150, batch_size: int = 50, use_gpu: bool = True, no_ans_boost: float = 0.0, return_no_answer: bool = False, top_k: int = 10, top_k_per_candidate: int = 3, top_k_per_sample: int = 1, num_processes: Optional[int] = None, max_seq_len: int = 256, doc_stride: int = 128, progress_bar: bool = True, duplicate_filtering: int = 0, use_confidence_scores: bool = True, proxies: Optional[Dict[str, str]] = None, local_files_only=False, force_download=False, use_auth_token: Optional[Union[str, bool]] = None, **kwargs, ,)
```

Comment:
why are here two trailing commas?

Reply:
Weird. The original method has one single trailing comma: https://github.com/deepset-ai/haystack/blob/3a6c8483596b624717bee20d5c7dad4227b183c9/haystack/nodes/reader/farm.py#L41-L64

Reply:
It also happens only here. I'll manually remove it and see what Black does.

Reply:
Ok it actually happens in a few places. I wonder if this is pydoc-markdown or black?

If you don't mind I'll leave like this for now. There's another PR open for pydoc-markdown which I will use to track this issue

Reply:
Yep, seems like it was pydoc-markdown: the diff comes from the documentation commit (https://github.com/deepset-ai/haystack/pull/2115/commits/472d14f0430e3826ba31c17a45e63764d371d650)